src/material/core/option/option.html

```diff
@@ -1,5 +1,6 @@
-<mat-pseudo-checkbox *ngIf="multiple" class="mat-mdc-option-pseudo-checkbox"
```

Comment:
Putting the checkmark on the end would look reasonable to me. I think it would make more sense if there was a API to configure if it's on the left or the right, like how we do it for MDC List. That's because it seems to me like it depends on the angular app's design and personal preference. WDYT?

Reply:
Okay, I would consider that out-of-scope of this PR, but I filed a feature request for it, so we can track it. #25984

Reply:
I like having the check at the start by default, matches the native select behavior

Reply:
To clarify what I meant above: I think that we should keep the checkmark at the end initially, but I'm fine with introducing an API to change it later on. I think that having it in the beginning looks weird, because it also introduces a whitespace before deselected options which almost makes it look like we were trying to center the text.

<img width="287" alt="image" src="https://user-images.githubusercontent.com/4450522/202661030-833f430e-46cf-4d76-967e-d1d6ede2bd8a.png">

Reply:
Ah yeah, I guess native select avoids this situation since there's always a selected element

Reply:
Done, minimal checkbox is at the end of the option.